docs/src/Box.doc.js

```diff
@@ -225,390 +247,487 @@ card(
 );
 
 card(
-  <Example
-    description={`
-    The [media object](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) is a common pattern for displaying data. What's interesting about this example is the use of \`flex\` to align the items. If you try changing the size of the \`Avatar\` or the number of lines of \`Text\`, both will stay aligned because they are center aligned.
-    Also, this is a good example of symmetrical padding. Try re-ordering the \`Avatar\` and the \`Box\` containing the text. You don't need to do any styling to keep the padding the same because it's symmetrical (\`paddingX\`). People often use something like \`marginStart\` here but in doing so you lose flexibility and have to change styling if you make changes to the structure.
-  `}
-    name="Example: Media object"
-    defaultCode={`
-<Box
-  alignItems="center"
-  direction="row"
-  display="flex"
-  marginStart={-1}
-  marginEnd={-1}
->
-  <Box paddingX={1}>
-    <Avatar name="chrislloyd" size="md" />
-  </Box>
-  <Box paddingX={1}>
-    <Text weight="bold">Chris Lloyd</Text>
-    <Text>joined 2 years ago</Text>
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description={`
+        Use Box as a building block when creating other components or layouts that do not rely on Flexbox. The included properties should cover any variations needed to create a diverse range of options.
```

Comment:
nit: "flexbox" shouldn't be capitalized, per MDN: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox#introducing_a_simple_example